css/index.css

```diff
@@ -111,7 +144,36 @@ p {
     align-items: center;
     justify-content: center;
 }
-
+.middle-content .boxes .box1 {
```

Comment:
good job using CSS selectors instead of adding new classes, this is more precise